src/test/java/com/google/cloud/tools/appengine/cloudsdk/serialization/CloudSdkVersionTest.java

```diff
@@ -56,6 +68,18 @@ public void testToString() {
   public void testEquals_differentSizes() {
     assertTrue(new CloudSdkVersion("0.1.0").equals(new CloudSdkVersion("0.1")));
     assertTrue(new CloudSdkVersion("0").equals(new CloudSdkVersion("0.0.0.0")));
+    assertFalse(new CloudSdkVersion("1.1").equals(new CloudSdkVersion("1.10")));
+  }
+
+  @Test
+  public void testEquals_preRelease() {
+    // TODO(alexsloan): implement and assert true semver comparisons, such that prerelease suffixes
```

Comment:
you probably don't want this todo in the test case. It should be in the non-test code.